pyproject.toml

```diff
@@ -31,7 +31,7 @@ classifiers = [
     "Intended Audience :: Other Audience",
     "License :: OSI Approved :: Apache Software License"
 ]
-include = ["IDENTITY"]
+include = ["volttron-platform-driver-0.2.0rc0-default-vip-id"]
```

Comment:
This is going to need to be dynamic if we are going to be updating the version info.  You will need to have it update when the deploy happens....

Reply:
So you need to update the .github actions for deploy development.

Reply:
Adding to Craig's comment, I think the action that needs to be updated is [here in deploy-pre-release](https://github.com/eclipse-volttron/volttron-platform-driver/blob/develop/.github/workflows/deploy-pre-release.yml#L101).